test_insertion.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import unicode_literals
+from insertion_sort import insertion_sort
+from random import shuffle
+
+
+def test_simple():
+    simple_list = [10, 7, 8, 9, 2, 15, 0, 5]
+    sorted_simple = insertion_sort(simple_list)
+    assert sorted_simple != simple_list
+    assert sorted_simple == sorted(simple_list)
```

Comment:
sorted(simple_list) is a cool way to test your sort with the built in. As long as you can guarantee the built in is working..